contracts/context-config/src/sys.rs

```diff
@@ -36,8 +36,12 @@ impl ContextConfigs {
         for (_, context) in self.contexts.drain() {
             drop(context.application.into_inner());
             context.members.into_inner().clear();
+            // let mut proxy = context.proxy.into_inner();
+            // proxy = AccountId::try_from("".to_string()).unwrap();
```

Comment:
```suggestion
            context.proxy.into_inner();
```

should suffice, I think